### DISCUSSION TITLE
如何解决中英文输入法带来的切换效率问题？

### DISCUSSION THREAD

**Discussion by Sealt** [BYJRK/DotNet-Discussions#199] [通用]
时而需要打中文时而需要打英文代码，想要打代码的时候出现中文输入法真的很影响效率和心情，UP是怎么解决这个问题的？

**Comment by BYJRK** [comment]
我不知道什么好办法😂其实我自己写代码一般是全英文，就有这方面的原因

**Comment by Sealt** [comment]
多方翻阅，发现微软拼音输入法有个不同窗口中英文乱跳的问题，是导致我切换体验极差的根本原因，微软多年来没改过这个问题，网友用脚本写了一个保证微软输入法默认中文的脚本，https://gist.github.com/maokwen/4d99f5c0aa2e7c0c114c708b03fb73ae
